tools-image/enki/cmd/build-iso.go

```diff
@@ -0,0 +1,126 @@
+package cmd
+
+import (
+	"fmt"
+	"os/exec"
+
+	"github.com/kairos-io/enki/pkg/action"
+	"github.com/kairos-io/enki/pkg/config"
+	"github.com/kairos-io/enki/pkg/utils"
+	v1 "github.com/kairos-io/kairos-agent/v2/pkg/types/v1"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"k8s.io/mount-utils"
+)
+
+// NewBuildISO returns a new instance of the buid-iso subcommand and appends it to
+// the root command. requireRoot is to initiate it with or without the CheckRoot
+// pre-run check. This method is mostly used for testing purposes.
+func NewBuildISO(root *cobra.Command, addCheckRoot bool) *cobra.Command {
```

Comment:
`addCheckRoot` is a bit strange. Does the command need root privileges or not? If yes, there is no reason to make this configurable, let's just always check. If not, the opposite.

Reply:
This is done with expectations for the future, i.e. pull-image may also have its niche here (pull-image, build-iso from that image) so I kept it abstrast on purpouse.

Also, this is just coming from the elemental-cli directly, no changes there.